tardis/montecarlo/montecarlo_numba/interaction.py

```diff
@@ -21,18 +21,16 @@
 def thomson_scatter(r_packet, time_explosion):
     """
     Thomson scattering — no longer line scattering
-    2) get the doppler factor at that position with the old angle
-    3) convert the current energy and nu into the comoving
-        frame with the old mu
-    4) Scatter and draw new mu - update mu
-    5) Transform the comoving energy and nu back using the new mu
+    \n2) get the doppler factor at that position with the old angle
+    \n3) convert the current energy and nu into the comoving frame with the old mu
+    \n4) Scatter and draw new mu - update mu
+    \n5) Transform the comoving energy and nu back using the new mu
```

Comment:
Considering code below, the numbered list should start from 1) instead of 2)

On a side note, we should never mention implementation details of function in its docstring instead they should be in code comments - docstrings are for specifying purpose of function. @KevinCawley you can leave it for now by just making numbering change.

Reply:
Ah oh, I meant:
```
1) get the ....
.
.
4) transform the ...
```
1st line is general description of function, not the 1st step of implementation.